### PR TITLE
#167190444 User can make comments to articles

### PR DESCRIPTION
**Pull request by encodedBicoding**
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[#167190444](https://www.pivotaltracker.com/story/show/167190444)
#### Screenshots (if appropriate)
N/A
#### Questions:
N/A

### CODE REVIEW

**Review by chiboycalix — APPROVED** [review]
LGTM

**Review by obayomi96 — APPROVED** [review]
LGTM

**Review by chuxmykel — APPROVED** [review]
Nice work on the eager loading bro. Looks good to me

File: server/db/models/article.js [hunk 1]
```diff
@@ -30,6 +30,9 @@ module.exports = (sequelize, DataTypes) => {
   });
   Article.associate = function (models) {
     Article.belongsTo(models.User, { foreignKey: 'id', onDelete: 'CASCADE' });
+    Article.hasMany(models.Comment, {
```
Comment (chuxmykel) [inline server/db/models/article.js:32]: Nice.

**Review by Hadeneekeh — APPROVED** [review]
LGTM

**Review by Lundii — APPROVED** [review]
LGTM

**Review by topseySuave — APPROVED** [review]

**Review by topseySuave — APPROVED** [review]

**Pull request merged**